### PR TITLE
CodeDocs repos link add

### PR DESCRIPTION
**Pull request by maddhruv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ Register at - https://hacktoberfest.digitalocean.com/
 
 **Links**
 
+| Name | Org/Developer | Link | Topic |
+| --- | --- | --- | --- |
+| JECRC-Pay | CodeDocsJECRC | https://github.com/CodeDocsJECRC/JECRC-Pay | Android, Java |
+| CodeDocs-App | CodeDocsJECRC | https://github.com/CodeDocsJECRC/CodeDocs-App | Android, Kotlin |
+| JECRC-Pay-API | CodeDocsJECRC | https://github.com/CodeDocsJECRC/JECRC-Pay-API | Node.js, MongoDB |
+
```
Comment (Man-Jain) [inline README.md]: Add Attendance Management Also with Tech Stack as Python, Flask And all thing for web

File: README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ Register at - https://hacktoberfest.digitalocean.com/
...
+
```
Comment (Man-Jain) [inline README.md]: Also we can add a Attendance Management api for the android app to use also built uaing flask

File: README.md (hunk 1 above)
Comment (maddhruv) [reply README.md]: we are not confirm with the tech stack as of now

Comment (Man-Jain) [reply README.md]: Okay, Then its fine

Comment (Man-Jain) [reply README.md]: We should put the links in some encapsulator (I dont know what it is called 😅  ). Also imo link should be the last column.

**Comment by akashlakhera** [comment]
I think we should make a different section for CodeDocs Repos because most of the people participating in Hacktoberfest are new and would want to do an Hello World PR first to get some interaction with github and open source.

Then if they are comfortable, they can try to help us on our live projects.

**Pull request merged**